benchmark/benchmark.js

```diff
@@ -16,6 +16,7 @@ var IMPLS = [];
 
 
 fs.readdirSync(IMPLS_DIRECTORY).sort().forEach(function (name) {
+  if (name === '.DS_Store') { return; }
```

Comment:
I could understand such file in .gitignore (don't know what it means), but placing such condition into code looks wrong.